### PR TITLE
Add picture on my fourth header

### PR DESCRIPTION
**Pull request by dinaris23**
Just added placeholder picture
fourth-header

### CODE REVIEW

**Comment by dinaris23** [comment]
1. Delete placeholder
2. Markup for fourth header
3. Add "img" folder and logo.png

**Comment by dinaris23** [comment]
Finished fourth header
54d8658914

File: app/styles/headers/_first.scss [hunk 1]
```diff
@@ -1,6 +1,3 @@
 // styles for first footer
 
-.header__first {
-	background-color: green;
-	height: 80px; 
-}
+
```
Comment (Vitaliy-Maznyi) [inline app/styles/headers/_first.scss:3]: why did you remove this styles?

File: app/styles/headers/_fourth.scss [hunk 2]
```diff
@@ -0,0 +1,65 @@
+.header__fourth {
+    max-width: 1180px;
+    width: 100%;
+    height: 100vh;
```
Comment (Vitaliy-Maznyi) [inline app/styles/headers/_fourth.scss]: please, don't use `100vh` for header height. It's always better when height of block depends on its content.

File: app/styles/headers/_fourth.scss [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+    color: #fff;
+    font-family: $font;
+    margin: 0 auto;
+    line-height: 1.5;
+    background: #262626 url("../img/iPhone-bg.png") no-repeat center;
+        &__wrapp {
+            width: 80%;
+            height: 80vh;
+            margin: 0 auto;
+            padding-top: 5%;
```
Comment (Vitaliy-Maznyi) [inline app/styles/headers/_fourth.scss]: please, don't use padding in percents

File: app/styles/headers/_fourth.scss [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+    width: 100%;
+    height: 100vh;
+    color: #fff;
+    font-family: $font;
+    margin: 0 auto;
+    line-height: 1.5;
+    background: #262626 url("../img/iPhone-bg.png") no-repeat center;
+        &__wrapp {
+            width: 80%;
+            height: 80vh;
```
Comment (Vitaliy-Maznyi) [inline app/styles/headers/_fourth.scss]: please, refactor height using paddings, because on huge display it will be probably broken down

File: app/styles/headers/_fourth.scss [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+
+        }
+        &__menu {
+            flex: 2;
+        
+        }
+
+        &__item {
+            display: inline-block;
+            margin-right: 5%;
```
Comment (Vitaliy-Maznyi) [inline app/styles/headers/_fourth.scss]: please, refactor margin and paddings with percents there and in other places

**Review by Vitaliy-Maznyi — DISMISSED** [review]

File: app/templates/index.html [hunk 6]
```diff
@@ -19,7 +19,14 @@ <h1>HELLO WORLD!</h1>
     	{% include "_header-2.html" %}
     </div>
     <div class="header header__third"></div>
+    <div class="header header__fourth">
```
Comment (pl4fun) [inline app/templates/index.html:22]: ogon'

**Review by pl4fun — APPROVED** [review]
